### ISSUE TITLE
Error: Invalid login: 534-5.7.14 responseCode: 534

### ISSUE THREAD

**Issue by AlexPro885** [nodemailer/nodemailer#998]
Hello! I use my account to send messages. When I start using the same code on a remote server, I get this error

`{ Error: Invalid login: 534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbsu
534-5.7.14 UOV711iivRHEfIeXzOmy2_wZbffsR9K3dxHBZ16f79wYlxpJXtjNTv9_57gP-bxO5jmsov
534-5.7.14 MAebgcVzoXLcepsY8z3iTXYlxX2kEXUOIpSHt4LmAJubm0v8uc3lv-GBKhhzBavRfkLMj7
534-5.7.14 W5WpHMlv5vgb_V_bjMfdba3ny3uivEgLOMB45JDhCrSMC07qkHL5CJ5h> Please log
534-5.7.14 in via your web browser and then try again.
534-5.7.14  Learn more at
534 5.7.14  https://support.google.com/mail/answer/78754 v1sm5499961eja.7 - gsmtp
    at SMTPConnection._formatError (/root/bochica/site-backend/node_modules/nodemailer/lib/smtp-connection/index.js:781:19)
    at SMTPConnection._actionAUTHComplete (/root/bochica/site-backend/node_modules/nodemailer/lib/smtp-connection/index.js:1516:34)
    at SMTPConnection._responseActions.push.str (/root/bochica/site-backend/node_modules/nodemailer/lib/smtp-connection/index.js:554:26)
    at SMTPConnection._processResponse (/root/bochica/site-backend/node_modules/nodemailer/lib/smtp-connection/index.js:940:20)
    at SMTPConnection._onData (/root/bochica/site-backend/node_modules/nodemailer/lib/smtp-connection/index.js:746:14)
    at TLSSocket.SMTPConnection._onSocketData (/root/bochica/site-backend/node_modules/nodemailer/lib/smtp-connection/index.js:189:46)
    at TLSSocket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
    at readableAddChunk (_stream_readable.js:265:11)
    at TLSSocket.Readable.push (_stream_readable.js:220:10)
  code: 'EAUTH',
  response:
   '534-5.7.14 <https://accounts.google.com/signin/continue?sarp=1&scc=1&plt=AKgnsbsu\n534-5.7.14 UOV711iivRHEfIeXzOmy2_wZbffsR9K3dxHBZ16f79wYlxpJXtjNTv9_57gP-bxO5jmsov\n534-5.7.14 MAebgcVzoXLcepsY8z3iTXYlxX2kEXUOIpSHt4LmAJubm0v8uc3lv-GBKhhzBavRfkLMj7\n534-5.7.14 W5WpHMlv5vgb_V_bjMfdba3ny3uivEgLOMB45JDhCrSMC07qkHL5CJ5h> Please log\n534-5.7.14 in via your web browser and then try again.\n534-5.7.14  Learn more at\n534 5.7.14  https://support.google.com/mail/answer/78754 v1sm5499961eja.7 - gsmtp',
  responseCode: 534,
  command: 'AUTH PLAIN' }`

**Comment by Kamikadze4GAME** [comment]
@AlexPro885 First of all read [README](https://github.com/nodemailer/nodemailer#im-having-issues-with-gmail), then lookup same issues and only after - create new Issue.

**Comment by andris9** [comment]
https://nodemailer.com/usage/using-gmail/

**Comment by voigtd** [comment]
Even after enabling less secure apps and completing the "Captcha Enable" challenge, the solutions mentioned (https://nodemailer.com/usage/using-gmail/) still don't work. The "Captcha Enable" seems to allow logins for a few hours but after that logins will start to be blocked again.

**Comment by mashkovtsevlx** [comment]
@voigtd Add 2FA to account and create APP password. Helped me.

**Comment by Rehan-05** [comment]
const sendResentEmail = async (email, subject, text) => {
     
  try {
    const transporter = nodemailer.createTransport({
        service: 'gmail',
        type: "SMTP",
        host: "smtp.gmail.com",
        port: 587,
        secure: false,
        auth: {
          user: 'your_Gmail', // generated ethereal user
          pass: 'your_Password', // generated ethereal password
        },
        // tls:{
        //   rejectUnauthorized:false
        // }
    });

    await transporter.sendMail({
        from: email, // sender address
        to: email,// list of receivers
        subject: subject,
        text: text,
    });

    console.log("email sent successfully");
    } catch (error) {
       console.log(error, "email not sent");
    }      
  };



And one thing more I have also Enabled the
 [“Less Secure”] (https://support.google.com/accounts/answer/6010255?hl=en) from Gmail account 
and also
 [“Captcha Enable”](https://accounts.google.com/b/0/displayunlockcaptcha).

 
Visit This : https://nodemailer.com/usage/using-gmail/

Thanks
